webformsasinput.md

```diff
@@ -0,0 +1,133 @@
+
+So. The one we want to understand is the one attached to the "Descriptive Metadata" field. Currently one named "Strawberryfield webform basewd input with inline rendering". There are other two. but lets start with this. Press on the Gear to the right on the same row.
+
+![Widget Inline Webform](../imgs/widget-inline-webform.jpg)
+
+AS you can see there are not too many options. But, the main, first Text input is an Autocomplete field that will resolve against your existing Webforms.
+So, guess what. If you want to use your own Webform to feed a SBF, what do you do? You type the name, let the autocomplete work, select the right Webform, maybe your own custom one, and the you press "Update". Once that is done you need to "Save" your Form Mode (hint, button at the bottom of the page).
+
+We wish life was that easy (and it will once we are done with refining Drupal's UI) but for now there are some extra things you need to do to make sure the Webform, your custom one, can speak JSON. The default one you get named also "Descriptive Metadata (descriptive_metadata)", same as the field, is already setup to be used. Means if you create a new Webform by Copying that one, you can start using it inmediately. But if you created one from scratch (Different tutorial) you need to setup some settings. 
+
+## Setting up a Webform to talk Strawberryfield
+
+Navigate to your Webform Managment form at `/admin/structure/webform`
+
+![Webform Managment](../imgs/webform-managment.jpg)
+
+If you already created a Webform (different tutorial on how to do that) you will see your own named one in that list. I created for the purpose of this documentation one named "Diego Test" (Hi, i'm Diego..) and on the most right Column, "Operations" you will haven an Drop Down Menu. On your own Webform row, press on "Settings".
+
+First time, this can be a little bit intimitading. We recommend going baby steps since the Webform Module is a very powerful one but also exposes you to a lot (and sometimes too many) options. Even more, if you are new to Webforms, we recomment you to copy the "Descriptive Metadata" Webform we provided first, and make small changes to it (starting by naming it your own way!) so you can see how that affects your workflow and experience, and how that interacts with the created metadata. The Webform Module provides testing and building capabilities, so you have a Playground there before actually ingesting ADOs. Copying it will also make all the needed settings for SBF interaction to be moved over, so your work will be much easier.
+
+But we know you did not do that (where is the fun there right?). So lets setup one from scratch.
+
+### General Settings
+
+![Webform General Settings](../imgs/webform-general-settings.jpg)
+Gist here is (look at the screenshot and copy the settings):
+
+- Disable Saving of Form Submissions. You won't need this form to generate a Native Webform Submission entry. 
```

Comment:
@DiegoPino Here it's not clear to me if we have to check or uncheck the option

Reply:
Ok, will reword :)